src/lsp/cobol_ptree/common.ml

```diff
@@ -30,3 +30,61 @@ type 'a nel = 'a NEL.t
 [@@deriving ord]
 
 let pp_nel ppe = NEL.pp ~fsep:"@ " ~fopen:"" ~fclose:"" ppe
+
+type permutation_duo = AB | BA
+[@@deriving ord]
```

Comment:
I think keeping only the last `deriving` is enough for `type … and … and …` declarations (but I may be wrong on that).